run_network_test.go

```diff
@@ -196,9 +295,16 @@ func TestRunPort(t *testing.T) {
 			testContainerName := fmt.Sprintf("nerdctl-test-nginx-%d", i)
 			base := testutil.NewBase(t)
 			defer base.Cmd("rm", "-f", testContainerName).Run()
-			pFlag := fmt.Sprintf("%s:%d:80", tc.listenIP.String(), tc.port)
-			connectURL := fmt.Sprintf("http://%s:%d", tc.connectIP.String(), tc.port)
+			pFlag := fmt.Sprintf("%s:%s:%s", tc.listenIP.String(), tc.hostPort, tc.containerPort)
+			connectURL := fmt.Sprintf("http://%s:%d", tc.connectIP.String(), tc.connectURLPort)
 			t.Logf("pFlag=%q, connectURL=%q", pFlag, connectURL)
+			if tc.assert != "ok" {
```

Comment:
This check should occur after creating `base.Cmd("run", "-d", ...)`

Reply:
done